chat.go

```diff
@@ -141,13 +160,20 @@ func (api *Client) PostEphemeralContext(ctx context.Context, channel, userID str
 }
 
 // UpdateMessage updates a message in a channel
-func (api *Client) UpdateMessage(channel, timestamp, text string) (string, string, string, error) {
-	return api.UpdateMessageContext(context.Background(), channel, timestamp, text)
+func (api *Client) UpdateMessage(channel, timestamp string, params UpdateMessageParameters) (string, string, error) {
```

Comment:
this breaks API compatibility, won't be accepted until we decide to do a batch of breaking changes.